app/src/hooks/useTransferProgress.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useRef, useState } from 'react'
+import { StoredTransfer } from '@/models/transfer'
+import { Direction } from '@/services/transfer'
+import { EstimatedTransferDuration } from './useTransfersEstimate'
+
+const getTransferProgress = (
+  transferDate: Date,
+  transferDirection: Direction,
+  bridgeStatus: EstimatedTransferDuration,
+) => {
+  let bridgeTimestamp = 0
```

Comment:
1. This is not a "timestamp", right? And we should include the units in the name and rename it for clarity

```suggestion
  let estimatedDurationMs = 0
```

2. Not sure if it's useful to have `bridgeStatus: EstimatedTransferDuration` as a parameter now since it will be a hardcoded value. So it would make more sense in my view to just reference it directly until we have plans to have a good API offering us this data dynamically.

3. With 2. in mind, I also think this `switch` logic should be moved to a separate function `getDurationEstimate(direction, estimatedDuration)` that would return the estimate in seconds. Then here this whole 13 lines of code becomes:

```
let estimatedDurationMs = getDurationEstimate(direction)
```